src/core/mount.c

```diff
@@ -1862,6 +1862,10 @@ static void mount_enumerate(Manager *m) {
                 }
 
                 (void) sd_event_source_set_description(m->mount_event_source, "mount-monitor-dispatch");
+
+                r = sd_event_source_set_ratelimit(m->mount_event_source, 1 * USEC_PER_SEC, 5);
+                if (r < 0)
+                        log_warning_errno(r, "Failed to enable rate limit for mount events, ignoring: %m");
```

Comment:
This shouldn't fail, so I think it should be treated the same as other failures above.